java/src/by/gsu/epamlab/task3/Purchase.java

```diff
@@ -0,0 +1,57 @@
+package by.gsu.epamlab.task3;
+
+public class Purchase implements Comparable<Purchase> {
+    public static final String NAME = "Bicycle";
+    public static final int PRICE = 100000;
+    private int quantity;
+    private int discount;
+    private WeekDay weekDay;
+
+    public Purchase(int quantity, int discount, WeekDay weekDay) {
```

Comment:
There is one more constructor needed. In which parameter of WeekDay is **int** 
Purchase(int quantity, int discount, int x)